test/assertions.js

```diff
@@ -279,3 +279,8 @@ should.Assertion.add('EncryptedSimpleCsv', function() {
   csv[4].should.eql([ '' ]);
 });
 
+should.Assertion.add('deepEqualInAnyOrder', function(expectedUnsorted) {
```

Comment:
I'm not totally sure whether Should.js requires this, but the other custom assertions here specify `this.params`.

Reply:
Maybe required?  https://shouldjs.github.io/#assertion-assert

I've added something.